libutils/file_lib.c

```diff
@@ -562,8 +562,15 @@ int safe_open_create_perms(
 #ifdef __MINGW32__
     // Windows gets off easy. No symlinks there.
     return open(pathname, flags, create_perms);
-#else // !__MINGW32__
-
+#elif __TERMUX__
```

Comment:
This should be `#elif defined(__TERMUX__)` AFAICT.

Reply:
so long as `__TERMUX__`, if present, evaluates to non-zero, which I think it does, then `#elif` is "ok" but I don't disagree that `#elif defined(__TERMUX__)` is more clear and resilient.

Reply:
https://gcc.gnu.org/onlinedocs/cpp/If.html#If

```
Identifiers that are not macros, which are all considered to be the number zero. This allows you to write #if MACRO instead of #ifdef MACRO, if you know that MACRO, when defined, will always have a nonzero value. 
```